zerver/openapi/zulip.yaml

```diff
@@ -14427,6 +14427,11 @@ paths:
                           Whether push notifications are enabled for this organization. Typically
                           `false` for self-hosted servers that have not configured the
                           [Mobile push notifications service](https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html).
+                      realm_push_notifications_enabled_end_timestamp:
+                        type: integer
+                        nullable: true
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
```

Comment:
I think we can describe this "If the server expects the realm's push notifications access to end at a definite time in the future, the time at which this is expected to happen. Mobile clients should use this field to display warnings to users when the indicated timestamp is near."